root/static/scripts/edit/components/PartialDateInput.js

```diff
@@ -188,6 +209,21 @@ const PartialDateInput = (props: Props): React$Element<'span'> => {
         type="text"
         {...dayProps}
       />
+      {props.uncontrolled ? null : (
+        <>
+          {' '}
+          <input
+            className="partial-date-parser"
+            disabled={disabled}
+            id={'id-' + field.html_name + '.partial-date-parser'}
+            name={field.html_name + '.partial-date-parser'}
+            placeholder={l('Enter date string to parse')}
```

Comment:
I'm not sure if "string to parse" is the best phrasing to use for the average user, but don't really have any better suggestions. :)

Maybe just something like "Paste a full date here"?

Reply:
"or paste a full date here"?

Reply:
That sounds good to me.